sources/packages/api/src/services/application/application.service.ts

```diff
@@ -1343,4 +1343,13 @@ export class ApplicationService extends RecordDataModelService<Application> {
       )
       .getOne();
   }
+  async getApplicationByApplicationNumberStudent(
+    applicationNumber: string,
+    student: Student,
+  ): Promise<Application> {
+    return this.repo.findOne({
```

Comment:
Can we restrict the _select_ here?
This method is also just to check if the application number exists, right? Maybe it can return a boolean and just d a _dummy select 1_.